assets/js/wcmp-frontend.js

```diff
@@ -438,48 +467,53 @@ jQuery(function($) {
     },
 
     /**
-     * Gets the current shipping method.
+     * Get the current shipping method without the shipping class.
      *
      * @returns {String}
      */
-    getSelectedShippingMethod: function() {
-      var shippingMethod = MyParcelFrontend.selectedShippingMethod;
+    getShippingMethodWithoutClass: function() {
+      var shippingMethod = MyParcelFrontend.getSelectedShippingMethod();
+      var indexOfSemicolon = shippingMethod.indexOf(':');
 
-      var overrides = [
-        /**
-         * This shipping method will have the suffix ":1", but this is not present in the array of selected shipping
-         * methods from the SETTING_DELIVERY_OPTIONS_DISPLAY setting.
-         */
-        'flat_rate',
+      shippingMethod = shippingMethod.substring(0, indexOfSemicolon === -1 ? shippingMethod.length : indexOfSemicolon);
 
-        /**
-         * For compatibility with Bolder Elements WooCommerce Table Rate Shipping, their format is different from the
-         * other shipping methods.
-         *
-         * @see https://bolderelements.net/plugins/table-rate-shipping-woocommerce/
-         */
-        'betrs_shipping',
-      ];
+      return shippingMethod;
+    },
 
-      overrides.forEach(function(override) {
-        if (shippingMethod.indexOf(override) === 0) {
-          shippingMethod = override;
-        }
+    /**
+     * Get the highest shipping class by doing a call to WordPress. We're getting it this way and not from the
+     *  highest_shipping_class input because that causes some kind of timing issue which makes the delivery options not
+     *  show up.
```

Comment:
Uitlijning van de command (spatie te veel voor `highest_shipping_class` en `show up` )

Reply:
Nee dat is de bedoeling. Zo komt het in de geformatteerde docs als 1 paragraafje te staan. Het is op alle andere plekken ook zo.